pytest/tests/sanity/rpc_missing_block.py

```diff
@@ -0,0 +1,103 @@
+import sys, time
+import pathlib
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / "lib"))
+
+from cluster import start_cluster
+from configured_logger import logger
+import utils
+from geventhttpclient import useragent
+
+nodes = start_cluster(
+    num_nodes=4,
+    num_observers=1,
+    num_shards=4,
+    config=None,
+    extra_state_dumper=True,
+    genesis_config_changes=[
+        ["min_gas_price", 0],
+        ["max_inflation_rate", [0, 1]],
+        ["epoch_length", 10],
+        ["block_producer_kickout_threshold", 70],
+    ],
+    client_config_changes={
+        0: {
+            "consensus": {
+                "state_sync_timeout": {
+                    "secs": 2,
+                    "nanos": 0
+                }
+            }
+        },
+        1: {
+            "consensus": {
+                "state_sync_timeout": {
+                    "secs": 2,
+                    "nanos": 0
+                }
+            }
+        },
+        2: {
+            "consensus": {
+                "state_sync_timeout": {
+                    "secs": 2,
+                    "nanos": 0
+                }
+            }
+        },
+        3: {
+            "consensus": {
+                "state_sync_timeout": {
+                    "secs": 2,
+                    "nanos": 0
+                }
+            }
+        },
+        4: {
+            "consensus": {
+                "state_sync_timeout": {
+                    "secs": 2,
+                    "nanos": 0
+                }
+            },
+            "tracked_shards": [0, 1, 2, 3],
+        },
+    },
+)
+
+nodes[1].kill()
```

Comment:
Why is killing node 1 necessary?

Reply:
In order to replicate a missing block state we set up nodes and then kill one. After this procedure many blocks will be considered as missing. This idea was proposed by @marcelo-gonzalez and it works great locally